packages/ibm-products/src/components/ConditionBuilder/ConditionBlock/ConditionBlock.js

```diff
@@ -66,12 +69,12 @@ const ConditionBlock = (props) => {
   };
 
   const { icon, type, config, label } = getCurrentConfig(property);
-  let ItemComponent;
-  //   if (type == 'custom') {
-  //     ItemComponent = config.component;
-  //   } else {
-  ItemComponent = property ? itemComponents[type] : null;
-  //}
+  let ItemComponent = null;
```

Comment:
lets just use a function here to define this. something like
```
getItemComponent = () => {}
...
itemComponent = getItemComponent();
```

Reply:
modified as advised